src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -659,11 +668,25 @@ private String uniqueName() {
     return prefix + suffix;
   }
 
-  private Metadata metadata() {
+  private Metadata newMetadata() {
+    Metadata metadata = new Metadata();
+    metadata.setItems(new ArrayList<Metadata.Items>());
+    return metadata;
+  }
+
+  private GoogleKeyPair congifureSSHKeyPair(Instance instance, String sshUser) {
```

Comment:
s/congifureSSHKeyPair/configureSSHKeypair
Do we need method headers?

Reply:
Not typically for private methods.